src-docs/src/views/card/card_layout.js

```diff
@@ -14,16 +14,17 @@ export default () => (
         layout="horizontal"
         icon={<EuiIcon size="xl" type={'logoBeats'} />}
         title={'Elastic Beats'}
-        description="Example of a card's description. Stick to one or two sentences."
+        description="This card adds uses an xl size icon and it works well in a horizontal layout."
         onClick={() => window.alert('Card clicked')}
       />
     </EuiFlexItem>
     <EuiFlexItem>
       <EuiCard
         layout="horizontal"
-        icon={<EuiIcon size="xl" type={'logoCloud'} />}
+        icon={<EuiIcon size="l" type={'logoCloud'} />}
+        titleSize="xs"
         title={'Elastic Cloud'}
-        description="Example of a card's description. Stick to one or two sentences."
+        description="This card uses a large size icon but also shrinks its titleSize to xs."
```

Comment:
To be consistent with the previous card...
```suggestion
        description="This card uses an `l` size icon but also shrinks the `titleSize` to `xs`."
```

Reply:
I don't know that these values need to be presented as EuiCode (i.e. my lazy addition of ticks) , main point here was the consistency of using a letter to denote the size (or spell out large and extra large in both cards).

Reply:
Yeah, I think I was being lazy too.. I'll just add some single quotes around them at least.